import-scripts/patients_vl.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import numpy as np
+import requests
+
+patients = np.genfromtxt('data/patients.csv', delimiter=',', dtype=None, encoding=None, skip_header=1)
+vl_obs = np.genfromtxt('data/vl_obs.csv', delimiter=',', dtype=None, encoding=None, skip_header=1)
+
+
+# SAVE PERSON OBJECT
+def savePerson(p):
+    person = {
+        "names": [
+            {
+                "givenName": "",
+                "familyName": ""
+            }
+        ],
+        "gender": "",
+        "birthdate": ""
+    }
+
+    person['names'][0]['givenName'] = p[1]
+    person['names'][0]['familyName'] = p[2]
+    person['gender'] = p[3]
+    person['birthdate'] = p[4]
+
+    try:
+        r = requests.post(
+            "https://openmrs-spa.org/openmrs/ws/rest/v1/person",
+            json=person,
+            auth=('admin', 'Admin123'),
+        )
+        r.raise_for_status()
+        print('PERSON SAVED SUCCESSFULLY')
+        return r.json()
+    except requests.exceptions.HTTPError as err:
+        raise SystemExit(err)
+
+
+# SAVE PATIENT OBJECT
+def savePatient(r):
+    patient = {
+        "person": r['uuid'],
+        "identifiers": [
+            {
+                "identifier": p[0],
+                "identifierType": "05a29f94-c0ed-11e2-94be-8c13b969e334",
+                "location": "58c57d25-8d39-41ab-8422-108a0c277d98",
+                "preferred": True
+            }
+        ]
+    }
+
+    try:
+        r = requests.post(
+            "https://openmrs-spa.org/openmrs/ws/rest/v1/patient",
+            json=patient,
+            auth=('admin', 'Admin123'),
+        )
+        r.raise_for_status()
+        print('PATIENT SAVED SUCCESSFULLY')
+        return r.json()
+    except requests.exceptions.HTTPError as err:
+        raise SystemExit(err)
+
+
+# SAVE ENCOUNTER OBJECT
+def savePatientEncounter(patient):
```

Comment:
The code in here should be modified in a couple of ways:

* The updated version of the code now outputs dates and times, so we should actually construct the `encounterDateTime` and `obsDateTime` from that.
* The obs concept can be one of two values, `856AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA` for numeric obs and `1305AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA` for coded obs. The values provided should correspond to those values (e.g., `944` for `856AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA`, `1301AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA` for `1305AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA`).